### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by hamdan286** [mupen64plus-ae/mupen64plus-ae#1063] [Q&A]
If I put the ROM file, it won't adjust and they don't tell me why?

**Comment by benjaJajala** [comment]
Cada vez que quiero  ejecutar un archivo rom , no me deja . ¿Por que?

**Reply by fzurita** [reply]
Trata de escanear por juegos de nuevo.

**Comment by Minizerker14** [comment]
So the game speed is never right how do i fix?

**Reply by fzurita** [reply]
What do you mean? Too fast or too slow? Does it happen with all games or only one game?
